### ISSUE TITLE
$orderby internals of array

### ISSUE THREAD

**Issue by dmigous** [Softmotions/ejdb#167]
For the following collection

```
{ _id: '55f7e3da512ce76200000000',
  name: 'aaa',
  links: 
   [ { price: 100 },
     { price: 7 },
     { price: 20 } ] }
{ _id: '55f7e3f6512ce76200000001',
  name: 'bbb',
  links: 
   [ { price: 200 },
     { price: 1 },
     { price: 2 } ] }
{ _id: '55f7ebc5512ce76200000003',
  name: 'ccc',
  links: [ { price: 1000 }, { price: 2000 } ] }
```

is it possible to sort this collection by lowest/highest `links.$.price`?

for lowest price I expect to have order

```
bbb (1) , aaa (7), ccc (1000)
```

for highest price I expect to have order

```
ccc (2000), bbb (200), aaa (100)
```

P.S. for me `db.arr.find({}, {$orderby: {'links.$.price': -1}})` doesn't work

**Comment by dmigous** [comment]
@adamansky I need your help :)

**Comment by adamansky** [comment]
Closed in favour to ejdb 2.0 https://github.com/Softmotions/ejdb/blob/master/README.md
